app/_data/docs_nav_gateway_2.8.x.yml

```diff
@@ -6,6 +6,7 @@
       absolute_url: true
     - text: Version Support Policy
       url: /support-policy
+      src: /gateway/support/index
```

Comment:
This doesn't work: https://deploy-preview-5321--kongdocs.netlify.app/gateway/2.8.x/support-policy/

The 2.8 docs isn't set to generate from `_src`. I wonder if you could this this method though:
https://deploy-preview-5321--kongdocs.netlify.app/contributing/single-sourced-versions/#single-source-specific-files (we haven't actually used it anywhere yet).

So  _I think_ it should be like this:

Top of the file:
```
product: gateway
release: 2.8.x
generate: true
assume_generated: false
```

Then set `generate: true` for the one item:
```
- text: Version Support Policy
  url: /support-policy
  src: /gateway/support/index
  generate: true
```

Reply:
That didn't work, I updated the file.